### PR TITLE
Add Date picker

### LINKED ISSUE #141: Move DatePicker component to Clockface
The DatePicker component which is being used in Data Explorer exists in InfluxDB repo needs to be moved to clockface for future use in Quartz.

### PR DESCRIPTION
**Pull request by mavarius**
Closes #141 

### Changes
Add DatePicker and DateRangePicker components.

### Screenshots
image

image

### Checklist
- [x] Updated documentation to reflect changes
- [x] Added entry to top of Changelog with link to PR (not issue)

### CODE REVIEW

File: .storybook/Story.scss [hunk 1]
```diff
@@ -304,7 +318,8 @@ body.sb-show-main {
     line-height: 1.55em;
     font-weight: 500;
 
-    b,strong {
+    b,
+    strong {
```
Comment (alexpaxton) [inline .storybook/Story.scss]: Reading this as "be strong", don't tell me how to live my life CSS!

File: src/Components/DatePicker/Base/DatePicker.tsx [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+  }
+
+  public render() {
+    const {dateTime, label, testID} = this.props
+
+    const date = new Date(dateTime || Date.now())
+
+    return (
+      <div className="date-picker" data-testid={testID}>
+        <Grid.Row>
```
Comment (alexpaxton) [inline src/Components/DatePicker/Base/DatePicker.tsx]: Does this component need `Grid` to work?

File: src/Components/DatePicker/Composed/DateRangePicker.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Date Range Picker
+
+This is a composed date picker that allows users to select a date range with a start and stop tim.
```
Comment (alexpaxton) [inline src/Components/DatePicker/Composed/DateRangePicker.md]: stop tim

File: src/Components/DatePicker/Base/DatePicker.tsx (hunk 2 above)
Comment (mavarius) [reply src/Components/DatePicker/Base/DatePicker.tsx]: Nope, removing

**Pull request merged**